packages/mgt-element/src/utils/Cache.ts

```diff
@@ -149,7 +149,23 @@ export class CacheService {
    * Clears all the stores within the cache
    */
   public static clearCaches() {
-    this.cacheStore.forEach(x => indexedDB.deleteDatabase(x.getDBName()));
+    this.cacheStore.forEach(async x => indexedDB.deleteDatabase(await x.getDBName()));
```

Comment:
I suspect we have a bug here that will be amplified with multiple accounts. The `cacheStore` map will only have a reference to a cache if that cache has been used at least once in the session. If we try to clear all stores without first switching to an account, not all stores will be cleared. Can we fetch all databases before trying to delete them?